website/settings/defaults.py

```diff
@@ -420,11 +420,11 @@ def parent_dir(path):
 RABBITMQ_PORT = os.environ.get('RABBITMQ_PORT', '5672')
 RABBITMQ_VHOST = os.environ.get('RABBITMQ_VHOST', '/')
 
-BROKER_URL = os.environ.get('BROKER_URL', 'amqp://{}:{}@{}:{}/{}'.format(RABBITMQ_USERNAME, RABBITMQ_PASSWORD, RABBITMQ_HOST, RABBITMQ_PORT, RABBITMQ_VHOST))
-BROKER_USE_SSL = False
+CELERY_BROKER_URL = os.environ.get('CELERY_BROKER_URL', 'amqp://{}:{}@{}:{}/{}'.format(RABBITMQ_USERNAME, RABBITMQ_PASSWORD, RABBITMQ_HOST, RABBITMQ_PORT, RABBITMQ_VHOST))
+CELERY_BROKER_USE_SSL = False
```

Comment:
this has to be reverted
https://github.com/celery/celery/blob/master/docs/userguide/configuration.rst#new-lowercase-settings /
http://docs.celeryproject.org/en/latest/userguide/configuration.html#std:setting-broker_use_ssl